app/Services/Spotify/OAuth/SpotifyTokenManager.php

```
@@ -0,0 +1,181 @@
+    /**
+     * Clear all tokens from cache (logout).
+     *
+     * @return void
+     */
+    public function clearTokens(): void
+    {
+        Cache::forget([
+            self::ACCESS_TOKEN_KEY,
+            self::REFRESH_TOKEN_KEY,
+            self::EXPIRES_AT_KEY,
+        ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'clearTokens'. (undefined)

(StaticAccess)